pkg/storage-init/storage-init.sh

```diff
@@ -11,15 +11,21 @@ chmod 700 $PERSISTDIR
 mkdir -p $CONFIGDIR
 chmod 700 $CONFIGDIR
 
-if CONFIG=$(findfs PARTLABEL=CONFIG) && [ -n "$CONFIG" ]; then
-    if ! fsck.vfat -y "$CONFIG"; then
-        echo "$(date -Ins -u) fsck.vfat $CONFIG failed"
-    fi
-    if ! mount -t vfat -o dirsync,noatime "$CONFIG" $CONFIGDIR; then
-        echo "$(date -Ins -u) mount $CONFIG failed"
-    fi
+# first look for a p9 config partition for override
```

Comment:
I am actually not a big fan of debug code being included in the default path in anything that we do

I honestly thought that you were looking for 9P to simply copy files in/out of the EVE -- but this is much more fundamental and something I'd like to avoid if possible.

Lets chat about this on slack some more before this gets merged. I can see a few options here (like this being commented out by default -- see the current commented out block at the bottom of the same script) Or maybe it should live in a debug package altogether.

Reply:
I don't necessarily disagree, but that code was all in there before anyways. If we want to change it, we should do that as a separate PR